ML/svm.md

```diff
@@ -0,0 +1,32 @@
+# Support Vector Machine (SVM)
+
+## Story
+
+The most common tasks in Machine Learning are classification, regression and outliers detection.  Among many algorithms, SVM is one which is used to solve the aforementioned tasks. You can use it to detect cancerous cells based on millions of images or you can use it to predict future driving routes with a well-fitted regression model.
+
+## Materials
+* [Support Vector Machines - Spiceworks](https://www.spiceworks.com/tech/big-data/articles/what-is-support-vector-machine/#:~:text=A%20support%20vector%20machine%20(SVM)%20is%20a%20machine%20learning%20algorithm,classes%2C%20labels%2C%20or%20outputs.)
+* https://www.ibm.com/topics/support-vector-machine
+* https://scikit-learn.org/stable/modules/svm.html
+* You're free to look at any other source for related materials. However, we strongly discourage you to copy/paste any code.
+
+## What you need to do
+
+### Part I
+* Familiarize yourself with basics of Machine Learning.
+* Explore Support Virtual Machines - its different types, practical use-cases and algorithmic analysis. You are also supposed to explore the sci-kit learn library for SVMs.
+* You need to make a presentation explaining what is SVM and how is it different from other popular Machine Learning algorithms (ex. kNNs, etc.) your thought process and how / why you choose a certain tool while solving this problem.
+
+### Part II
+* Pick a dataset (from ex. [Kaggle](https://www.kaggle.com/datasets)).
+* Write a code, using sci-kit learn (svm), to perform a non-linear classification. However, keep in mind that you will be required to explain the working of the algorithm you have used.
```

Comment:
How about writing svm from scratch (using numpy and pandas) ?

Reply:
done